src/views/AddItem.jsx

```diff
@@ -85,7 +86,7 @@ export function AddItem({ listToken, itemList, setData }) {
 								value="14"
 								onChange={handleTime}
 							/>
-							Kind of Soon (14 days)
+							Kind of Soon (7 ~ 30 days)
```

Comment:
It bothers me that in the first case we use the max value for the calculation, in the third case it is the min value (30), and in the second case it is 14, which is closer to the the min value to 7.

May be it would be easier to change only the description in the list view to _soon 7 days, kind of soon 7-14, not so/that soon 14-30 and inactive over 30_  so that only max values are used in the calculation?

Reply:
I mean that 60 days out of nowhere in one of the last tickets was really confusing, because we used only 7, 14, 30 before